docs/conf/portland/2023/sponsors/prospectus.rst

```diff
@@ -39,32 +36,52 @@ Why Sponsor
 Sponsorship Packages
 ====================
 
-The following options are suggested sponsorship levels. We are happy to discuss adjustments and custom packages.
+We have broken our sponsorships based on what it is you're trying to get out of the sponsorship.
+We find that many sponsors have two primary goals in mind:
+
+* Hiring
+* Product promotion
 
 Second Draft
 ------------
 
-The **Second Draft** package is great for companies looking to hire or to promote a product.
+The **Second Draft** package gives you visibility on the primary conference website and communications.
+It's great for sending folks to the conference,
+and showing support for the community.
+
+Benefits
+~~~~~~~~
 
 - Three (3) tickets_
-- Two (2) featured job postings on our Job Board, also promoted in our newsletter ({{ newsletter_subs }} subscribers)
-- A short description (250 characters) and logo of your company on the conference website
-- A table at our **virtual job fair**
+- A small logo & short text content (250 characters) on the conference website
 - Name included in welcome announcement in email newsletters and social media
 
 The **Second Draft** package costs **{{sponsorship.second_draft.price}}**.
 
 Publisher
 ---------
 
-The **Publisher** package is great for sending a team and getting to know the community.
+The **Publisher** package gets you a larger online presence and a visible on-site presence, either at the job fair or a sponsored workshop.
```

Comment:
This isn't super clear. Are they providing a workshop or sponsoring an existing one.